ui/src/app/settings/components/repos-list/repos-list.tsx

```diff
@@ -40,6 +40,7 @@ export interface NewHTTPSRepoParams {
     project?: string;
     forceHttpBasicAuth?: boolean;
     enableOCI: boolean;
+    insecureHttpOnly: boolean;
```

Comment:
Around line 308 of this file, it should show an icon for the repo type.  Unfortunately, the change should be in the argo ui repo, so will need to create a PR there. See: 

https://github.com/argoproj/argo-ui/blob/master/src/styles/argo-icon.scss and the icons folder there, that should contain the new oci svg icon.

Reply:
@keithchong that's a good point! I'm not much of a designer though, so would need help from someone to create the icon...